xc/common/utils/prjxray_create_synth_tiles.py

```diff
@@ -16,11 +16,14 @@ def map_tile_to_vpr_coord(conn, tile):
     c.execute("SELECT pkey FROM phy_tile WHERE name = ?;", (tile, ))
     phy_tile_pkey = c.fetchone()[0]
 
+    c.execute("SELECT pkey FROM tile_type WHERE name = 'NULL'")
+    null_tile_type_pkey, = c.fetchone()
+
     # It is expected that this tile has only one logical location,
     # because why split a tile with no sites?
     c.execute(
-        "SELECT tile_pkey FROM tile_map WHERE phy_tile_pkey = ?",
-        (phy_tile_pkey, )
+        "SELECT tile_pkey FROM tile_map WHERE phy_tile_pkey = ? AND tile_type_pkey != ?",
```

Comment:
tile_type_pkey is part of the phy_tile table, so you need to join tile_map and phy_tile on tile_map.phy_tile_pkey = phy_tile.pkey, and then you can filter like this with a WHERE clause on "phy_tile.tile_type_pkey"